lib/linguist/languages.yml

```diff
@@ -5066,6 +5066,15 @@ Shen:
   tm_scope: source.shen
   ace_mode: text
   language_id: 348
+Sieve:
+  type: programming
+  tm_scope: source.sieve
+  ace_mode: text
+  extensions:
+  - ".sieve"
+  codemirror_mode: sieve
+  codemirror_mime_type: application/sieve
+  language_id: 208976687
```

Comment:
I see you've not specified a colour for the language. Is this intentional? Without an explicit colour, the language will show up using the default `#cccccc` on some views like the language bar and with no colour at all on others like the repo listing.

Reply:
[Sieve's logo](http://sieve.info/lib/tpl/sieveinfo/user/logo.png) is grey, so it's not like `#ccc`  would be out-of-place.

Reply:
I'm happy with that is peeps are.

Reply:
I don't know of any particular color associated with Sieve. The aforementioned Logo isn't very colorful either.

Reply:
If we go with the grey color, it might be good to set it explicitly. That way, if we later add a test to check that all languages have a color, we won't need to dig through the comments in this PR :-)

Reply:
Assuming, of course, that we finally get an answer to the million-dollar question GitHub's staff have yet to reveal the answer to.

I'm talking, of course, about that agitating colour proximity requirement. 😉